### ISSUE TITLE
Combat orders,etc. do not seem to work unless you repack NPC

### ISSUE THREAD

**Issue by KevinHan522** [P3pp3rF1y/AncientWarfare2#448]
Every once in a while, the combat NPCs seem to completely stop doing what they were assigned to do whether it was to guard or etc.  The biggest problem is they do not follow their combat orders after a wjo;e.  They return to normal after repacking and placing but this is a massive pain.  They seem to go to this state after veering off their path for a long enough time.

**Comment by GotoLink** [comment]
Does it happen during rain / night ?
Do they have upkeep orders too ?

**Comment by KevinHan522** [comment]
I know it normally happens at night.  Haven't seen if it happens in rain yet.

They do not have upkeep orders.  I'll try that.

**Comment by Rosomah** [comment]
The same thing happened to me. After repacking and placing they start following the orders again. For me, it happened when it got dark and enemy mobs spawned. The guards were patrolling around an area and started attacking the enemy mobs. I saw them getting far away from the town hall so I went to bed so I wouldn't lose too many soldiers and when all the enemy mobs were gone, only on of the soldiers returned to his patrol duty, the others were just standing there as if they had no order equipped.
